lib/src/components/top-bar/TopBar.test.tsx

```diff
@@ -17,17 +17,45 @@ const ExampleTopBar = () => (
 )
 
 describe('TopBar component', () => {
-  it('renders', async () => {
+  it('renders without size prop', async () => {
```

Comment:
No need for the `without size prop` bit here. We do at least one snapshot with the default values
```suggestion
  it('renders', async () => {
```

Reply:
thanks will make this change